src/main/java/net/sf/jabref/model/database/BibDatabaseContext.java

```diff
@@ -154,18 +170,18 @@ public boolean isBiblatexMode() {
     * @param fieldName The field type
     * @return The default directory for this field type.
     */
-    public List<String> getFileDirectory(String fieldName, FileDirectoryPreferences preferences) {
-        List<String> fileDirs = new ArrayList<>();
+    public List<String> getFileDirectories(String fieldName, FileDirectoryPreferences preferences) {
+        List<String> fileDirs = new ArrayList<>(4);
```

Comment:
Seems like it was an artifact from my previous implementation approach.. It is no longer necessary.